src/shared/src/types/change-request-types.ts

```diff
@@ -129,6 +126,11 @@ export interface ProjectProposedChangesCreateArgs {
 }
 
 export interface WorkPackageProposedChangesCreateArgs {
+  name: string;
+  status: WbsElementStatus;
+  projectLeadId: number;
+  projectManagerId: number;
+  links: { url: string; linkTypeName: string }[];
```

Comment:
since these are common fields, you can keep them on WBSProposedChangesCreateArgs and just have ProjectProposedChangesCreateArgs and WorkPackageProposedChangesCreateArgs extend WBSProposedChangesCreateArgs